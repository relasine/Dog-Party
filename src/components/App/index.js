import React, { Component } from 'react';
import './App.css';

import Footer from '../Footer';
import DogsContainer from '../DogsContainer';

class App extends Component {
  constructor() {
    super()

    this.state = {
      category: 'All Dogs'
    }
  }

  render() {
    return (
      <div className="App">
        <DogsContainer category={this.state.category}/>
        <Footer />
      </div>
    );
  }
}

export default App;
