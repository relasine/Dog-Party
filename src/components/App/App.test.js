import React from 'react';
import ReactDOM from 'react-dom';
import App from './index.js';
import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('should match the snapshot', () => {
  const wrapper = shallow(<App />);

  expect(wrapper).toMatchSnapshot();
});