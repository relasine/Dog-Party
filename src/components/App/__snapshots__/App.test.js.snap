// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<div
  className="App"
>
  <Footer />
</div>
`;
